src/typegoose-core.module.ts

```diff
@@ -109,8 +110,7 @@ export class TypegooseCoreModule implements OnModuleDestroy {
         }
         return array;
       }, []).forEach((modelKey) => {
```

Comment:
```suggestion
      }, []).forEach(deleteModel)
```

pass the function, so you dont need to make this extra scope - i dont know if this is wanted by @kpfromer

Reply:
ok